### DISCUSSION TITLE
Light-Mode for low-spec computers

### DISCUSSION THREAD

**Discussion by yendri404** [ppy/osu#17799] [Ideas] [answered]
The **"original"** osu! was caracterizade of be a _"low-spec"_ game, but yeah i kow what lazer it's better on perfomrance but, some times playing on my PC gets very lagged, _(my PC is a potato with ram lol)_. So i think it is a good idea add a light-mode, when you activate that the most of visual effects become disabled to make the game most light, and of course i know what there things to add more importants than this idea, anyway thanks for take a time for reading this

**Comment by peppy** [comment, marked as answer]
"Visual effects" in lazer do not cause any kind of slowness you may be experiencing. We will continue to work on optimisations and you will find that once we are done the game will run better than stable.

Please do provide your system specs for reference.

**Reply by yendri404** [reply]
thank you for answer about my simple idea!, and my PC its:

(sorry for my bad English I'm learning English)

4 GB RAM
1.45 GHz on 4 Cores with a AMD sempron
without graphic card
windows 10 20H2
(yeah my Pc is a potato with cables but I play on this xD)

and very thank you so much for your time!

**Reply by peppy** [reply]
For now I'd highly recommend staying on osu! (stable) with those specs. But maybe one day this will change, with further optimisation.

**Comment by pranshuthegamer** [comment]
@peppy what are some things which you think could be major performance bottlenecks?
